cmd/ocm-backplane/login/login.go

```diff
@@ -405,3 +412,17 @@ func listNamespaces(clusterID, clusterName string, isHostedControlPlane bool) ([
 
 	return nsList, nil
 }
+
+// getBackPlanehost returns the DNS/CNAME resolution of the ocm backplane URL
+func getbackplanehost(backplaneURL string) (string, error) {
```

Comment:
Function names should ideally be written in CamelCase.

```suggestion
func getBackplaneHost(backplaneURL string) (string, error) {
```

Reply:
Changes applied in the e0989d47092e5c3e3de08fbb05fca5059c769ee8